### PR TITLE
Apply Enum Mappings to most enum settings

### PR DESCRIPTION
**Pull request by carlos-zamora**
## Summary of the Pull Request
Builds off of #8086 to introduce enum mappings for most of the remaining enum settings. This excludes the following two settings:
- FontWeight: we accept a number and an enum. So we'll probably need some special logic for this one (Kayla's working on this one)
- BackgroundImageAlignment: this is one setting stored as two enums. We'll need some special logic here too.

This also removes "Globals" files, a relic from the hackathon. We decided to split this up into all the other pages, and forgot to remove this at some point.

## References
#1564 - Settings UI Epic

## Validation Steps Performed
These become validated when we load the page with setting on it. Since there's a known issue right now that we can't load every available page (yet), I've only been able to validate that this works on the loadable pages.

If a setting was added, but we didn't update the resw properly, we get an error at runtime. So this helps a ton with debugging!

### CODE REVIEW

File: src/cascadia/TerminalSettingsEditor/Profiles.cpp [hunk 1]
```diff
@@ -1,20 +1,120 @@
...
+        auto bellStyleMap = EnumMappings::BellStyle();
+        for (auto [key, value] : bellStyleMap)
+        {
+            auto enumName = LocalizedNameForEnumName(L"Profile_BellStyle", key, L"Content");
+            auto entry = winrt::make<EnumEntry>(enumName, winrt::box_value<BellStyle>(value));
+            _BellStyles.Append(entry);
+
+            // Initialize the selected item to be our current setting
+            if (value == Profile().BellStyle())
+            {
```
Comment (DHowett) [inline src/cascadia/TerminalSettingsEditor/Profiles.cpp:114]: i mean, _this seems insane_.

File: src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw [hunk 2]
```diff
@@ -251,7 +251,8 @@
     <value>Launch size</value>
   </data>
   <data name="Globals_LaunchSize.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
-    <value>Defines whether the terminal will launch in a window, maximized, or full screen.</value>
+    <value>Defines whether the terminal will launch as maximized, full screen, or in a window. Setting this to "Focus" is equivalent to launching the terminal in the "Default" mode, but with the focus mode enabled. Similar, setting this to "MaximizedFocus" will result in launching the terminal in a maximized window with the focus mode enabled.</value>
```
Comment (DHowett) [inline src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw:254]: THE LOC FREEZE WAS THREE WEEKS AGO COME ON LOL

Comment (DHowett) [reply src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw:254]: also, the wording is wrong. `Maximized Focus` should be two words.

I don't think we're doing it right if we just copy the entire json schema field description into a freakin' tooltip, but it's what we've got today.

Comment (carlos-zamora) [reply src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw:254]: My bad. #7873 went in and I had no clue. I'll pull all the resw changes into a separate PR to get enough time for localization.

Comment (carlos-zamora) [reply src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw:254]: Moved all the localization stuff over to #8116. Removing this stuff from this PR.

**Comment by carlos-zamora** [comment]
I'm actually going to close this PR.
- localization stuff got moved to #8116 
- removing globals is going to be its own pr
- @leonMSFT will handle binding all of these enum settings (since he's making macros for them anyways)

**Pull request closed without merging**